snapshot/containerd/content.go

```diff
@@ -103,7 +103,7 @@ var _ content.Store = &nsFallbackStore{}
 func (c *nsFallbackStore) Info(ctx context.Context, dgst digest.Digest) (content.Info, error) {
 	info, err := c.main.Info(ctx, dgst)
 	if err != nil {
-		if errdefs.IsNotFound(err) {
+		if cerrdefs.IsNotFound(err) {
```

Comment:
I wonder if this alway works; some implementations appear to not be returning a typed error. For example; https://github.com/moby/buildkit/blob/59a3ce5dd63acdbdb2d2d8b58dbade3389660dd0/cache/remotecache/v1/chains.go#L130-L136

Reply:
I don't think these are the implementations for the `nsFallbackStore`